src/ActionModal/components/SendModal.vue

```diff
@@ -11,6 +11,7 @@
     :notify-message="notifyMessage"
     feature-flag="send"
     :disabled="sendingNgm"
+    :terra-tax="getTerraTax(false)"
```

Comment:
let's not make this network specific and use `chain-applied-fees`

Reply:
ah I see you use the tax in different ways. then make fees 0? doe we need to send the terra tax or is applied on chain?